adapter/Cargo.toml

```diff
@@ -19,8 +19,8 @@ eth_checksum = "0.1.1"
 ethabi = { git = "https://github.com/samparsky/ethabi", branch = "graph-patches" }
 tiny-keccak = "1.5"
 parity-crypto = { version = "0.4.2", features = ["publickey"] }
-ethstore = { version = "0.2.1", git = "https://github.com/paritytech/parity-ethereum"}
-ethkey = { version = "0.4.0", git = "https://github.com/paritytech/parity-ethereum"}
+ethstore = { version = "0.2.1", git = "https://github.com/elpiel/parity-ethereum"}
+ethkey = { version = "0.4.0", git = "https://github.com/elpiel/parity-ethereum"}
```

Comment:
why the change here?

Reply:
does it sync faster?

Reply:
faster - yes
fast build - not so much it still takes long time, but it's better.
I tried looking into caching dependencies, but not much luck on that.
@samparsky sent me a link, so I will check it if it's possible to use cache for the **build**